service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -305,13 +310,41 @@ void shouldNotifyRepresentativesOfCMOIssued() throws Exception {
                 eq(expectedCMOIssuedNotificationParametersForRepresentative), eq("12345"));
         }
 
+        @Test
+        void shouldNotifyAdminOfCMOReadyForJudgeReview() throws Exception {
+            CallbackRequest callbackRequest = callbackRequest();
+            CaseDetails caseDetails = callbackRequest().getCaseDetails();
+
+            given(hmctsCourtLookupConfiguration.getCourt(LOCAL_AUTHORITY_CODE))
```

Comment:
Do we need to do this given and the one below as well, I thought that we created some configuration beans in the so we didn't need to do this